backend/src/middleware/handleHtmlContent/handleContentData.js

```diff
@@ -0,0 +1,69 @@
+import extractMentionedUsers from './notifications/extractMentionedUsers'
+import extractHashtags from './hashtags/extractHashtags'
+
+const notify = async (postId, idsOfMentionedUsers, context) => {
```

Comment:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-07-01T22:33:10Z" title="Tuesday, July 2nd 2019, 12:33:10 am +02:00">Jul 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,69 @@
+import extractMentionedUsers from './notifications/extractMentionedUsers'
+import extractHashtags from './hashtags/extractHashtags'
+
+const notify = async (postId, idsOfMentionedUsers, context) => {
```


This part should be centralized in some notification handler - certainly not necessary to do this in this PR - but might be wise to open an Issue for that

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-02T06:17:00Z" title="Tuesday, July 2nd 2019, 8:17:00 am +02:00">Jul 2, 2019</time>_

---


I do not understand, what exactly you mean with a notification handler. @ulfgebhardt 
When we come to mentions in Comments, we have to do something about that, may be.

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-07-02T14:41:19Z" title="Tuesday, July 2nd 2019, 4:41:19 pm +02:00">Jul 2, 2019</time>_

---


i am talking about the `notify` function. We should centralize everything which has to do with notifications. And as I stated - not necessary in this PR